### PR TITLE
Migrate to modern Android Studio project structure

### PR DESCRIPTION
**Pull request by z3ntu**
Also rename package to community.fairphone.mycontacts

### CODE REVIEW

**Review by Roboe — APPROVED** [review]
LGTM

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        debug {
+            applicationIdSuffix ".debug"
+        }
+    }
+}
+
+dependencies {
+    api 'com.android.support:support-v4:21.0.3'
+    api 'com.android.support:appcompat-v7:21.0.3'
+    api 'com.googlecode.libphonenumber:libphonenumber:5.5'
```
Comment (z3ntu) [inline app/build.gradle]: @Roboe why `api`? `implementation` is used for these things.

Comment (Roboe) [reply app/build.gradle]: Oh, c'mon, I understood it the other way some days ago (`api` being more restrictive than `implementation`) (It also has more semantic sense, IMO: "I'll just use the api of this lib").

You are right, I'll drop that commit.

**Comment by Roboe** [comment]
Fixed, :+1:

**Comment by z3ntu** [comment]
LGTM now.

**Pull request merged**